### PR TITLE
feat: Small fixes & improvements for readme utility

### PR DESCRIPTION
**Pull request by AlexanderSehr**
## Description

- Added a resolve path to script to address users using unexpected relative paths that conflict with the rest of the logic
- Fleshed out the required parameter check (dedicated function and more meaningful parameter names)
- Small typo fix in a test case

> Note: Re-ran module generations that caused no issues

### CODE REVIEW

File: .github/workflows/avm.res.kubernetes-configuration.extension.yml
Comment (eriqua) [inline .github/workflows/avm.res.kubernetes-configuration.extension.yml:1]: :ghost:

File: avm/utilities/tools/Set-AVMModule.ps1 [hunk 1]
```diff
@@ -76,21 +76,19 @@ function Set-AVMModule {
         [string] $ReadMeScriptFilePath = (Join-Path (Get-Item $PSScriptRoot).Parent.FullName 'pipelines' 'sharedScripts' 'Set-ModuleReadMe.ps1')
     )
 
-    # Load helper scripts
-    . (Join-Path $PSScriptRoot 'helper' 'Set-ModuleFileAndFolderSetup.ps1')
```
Comment (eriqua) [inline avm/utilities/tools/Set-AVMModule.ps1:80]: Is it intended to be removed? This change was introduced recently, I wonder if we missed a pull from latest

Comment (AlexanderSehr) [reply avm/utilities/tools/Set-AVMModule.ps1:80]: Hell no. Damn :D

Comment (AlexanderSehr) [reply avm/utilities/tools/Set-AVMModule.ps1:80]: Undone

File: avm/utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1 [hunk 2]
```diff
@@ -990,6 +990,56 @@ function ConvertTo-FormattedBicep {
     return $commentedBicepParams
 }
 
+<#
+.SYNOPSIS
+Based on the provided parameter metadata, determine whether the parameter is required or not
```
Comment (eriqua) [inline avm/utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1:995]:
```suggestion
Based on the provided parameter metadata, determine whether the parameter is required or not.
```

**Review by eriqua — APPROVED** [review]
:ok:

**Pull request merged**